test.py

```diff
@@ -0,0 +1,8 @@
+import backtester
+
+cont = backtester.Controller('data/EUR_USD_H1.csv')
+cont.loadData()
+print(cont.dt)
+cont.incrementTime()
+print(cont.dt)
+print(cont.returnClose())
```

Comment:
We need to consider writing this out into a test framework, so rather than test.py we should create a bunch of files and what they are testing.

Reply:
I've made all attributes private.